Source/Scene/ModelInstance.js

```diff
@@ -0,0 +1,80 @@
+/*global define*/
+define([
+        '../Core/Color',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/Matrix4'
+    ], function(
+        Color,
+        defaultValue,
+        defined,
+        defineProperties,
+        Matrix4) {
+    "use strict";
+    
+    /**
+     * A model instance, contained within a ModelInstanceCollection
+     * 
+     * @private
+     */
+    var ModelInstance = function(options, collection, index, pickPrimitive) {
```

Comment:
I feel like this should be combined or somehow inherited from the same thing as `BatchedModel`.  I don't have a concrete idea yet, but once we do point clouds or vector data, we should be able to flush it out.